forgotPassword.php

```diff
@@ -0,0 +1,36 @@
+<?php session_start(); ?>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>13th Annual Alumni Meet &middot; Forgot Password</title>
+    <?php include 'topHead.php'; ?>
+  </head>
+  <body>
+    <?php include "navbar.php"; ?>
+    <div class="container">
+      <h2 align="center">Forgot Password</h2>
+      <?php
+      if (isset($_GET["alert"])) {
+        echo "<div class='alert alert-danger alert-dismissible'>";
+        echo "<button type='button' class='close' data-dismiss='alert' aria-label='Close'>";
+        echo "<span aria-hidden='true'>&times;</span>";
```

Comment:
@icyflame 
This is a simple dismissible alert, for displaying all the GET data.
After this PR get merged i will create that alert.php and include everywhere so that display alert gets easy task.